src/Workspaces/Remote/ServiceHub/Shared/ClientDirectStream.cs

```diff
@@ -37,6 +37,10 @@ private class ClientDirectStream : Stream
             // 4KB buffer size
             private const int BUFFERSIZE = 4 * 1024;
 
+            private const int ConnectTimeout = 1;
+            private const int ConnectRetryIntervalMs = 20;
```

Comment:
:bulb: Better to declare both of these as `private static readonly TimeSpan` instead of `const int`.

Reply:
sure

Reply:
Connect doesn't accept timespan so, I renamed it to ConnectWithoutTimeout.